### PR TITLE
Update Logs docs

### PR DESCRIPTION
**Pull request by solepano**
- Add link to new Logs query syntax page
- Add reference to logs extensions
- Remove reference to deprecated Auth0 Logs Processor repo

### CODE REVIEW

File: articles/logs/index.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ useCase:
 
 # Logs
 
-Using the [Dashboard](${manage_url}), you can pull log data on actions performed by administrators using the Dashboard, and authentications made by your users.
+Using the [Dashboard](${manage_url}/#/logs) or the [Management API logs endpoint](/api/v2#!/Logs/get_logs), you can pull log data on actions performed by administrators using the Dashboard, operations performed via the Management API and authentications made by your users.
```
Comment (machuga) [inline articles/logs/index.md]:
```suggestion
Using the [Dashboard](${manage_url}/#/logs) or the [Management API logs endpoint](/api/v2#!/Logs/get_logs), you can pull log data on actions performed by administrators using the Dashboard, operations performed via the Management API, and authentications made by your users.
```

File: articles/logs/query-syntax.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+- `user_name`
+- `connection`
+- `client_name`
+- `type`
+- `ip`
+- `log_id`
+- `description`
+
+### Exact matching
```
Comment (machuga) [inline articles/logs/query-syntax.md]:
```suggestion
### Exact Matching
```

**Review by d60402 — DISMISSED** [review]
Instead of "A date will always be converted to a zeroed-out state if all **fields** are not **supported**", can we change this to "A date will always be converted to a zeroed-out state if all **values** are not **provided**"?

File: articles/logs/query-syntax.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  - query-syntax
+contentType:
+  - reference
+useCase:
+  - manage-logs
+---
+
+# Logs Query Syntax
+
+The [Management API logs endpoint](/api/v2#!/Logs/get_logs) allows to retrieve logs either by checkpoint (last retrieved log id) or by search criteria. We highly recommend to use the checkpoint approach to export logs to the external system of your choice and perform any search or analysis there, as logs stored in our system are subject to [the retention period](/logs#how-long-is-log-file-data-available). You can use any of the [Export Auth0 logs to an external service](/extensions#export-auth0-logs-to-an-external-service) extensions to export the logs to the system of your choice (like Sumo Logic, Splunk or Loggly).
```
Comment (d60402) [inline articles/logs/query-syntax.md]: "...allows **you** to retrieve..."

File: articles/logs/index.md [hunk 4]
```diff
@@ -47,14 +63,54 @@ Enterprise | 30 days
 If you would like to store log data longer than the time period offered by your subscription plan, we recommend you use the [Management API feature that allows you to retrieve the relevant data](api/management/v2#!/Logs/get_logs). Once you've retrieved your data, you can:
 
 * Store the data yourself
-* Send the data to an external service such as Splunk (consider using the [Auth0 Logs to Splunk Extension](/extensions/splunk))
+* Send the data to an external service. You can install and configure an Auth0 Extension in order to export logs automatically to another provider, like Sumo Logic or Loggly. For a list of available providers and detailed steps to configure each, see [Export Auth0 logs to an external service](/extensions#export-auth0-logs-to-an-external-service).
+
+## Retrieving logs from the Management API
```
Comment (machuga) [inline articles/logs/index.md:69]:
```suggestion
## Retrieving Logs from the Management API
```

File: articles/logs/index.md [hunk 5]
```diff
@@ -47,14 +63,54 @@ Enterprise | 30 days
...
+::: info
+We highly recommend using [the checkpoint approach](/logs#get-logs-by-checkpoint) to export logs to the external system of your choice and perform any search or analysis there, as logs stored in our system are subject to [the retention period](/logs#how-long-is-log-file-data-available). You can use any of the [Export Auth0 logs to an external service](/extensions#export-auth0-logs-to-an-external-service) extensions to export the logs to the system of your choice (like Sumo Logic, Splunk or Loggly).
+:::
+
+If you would like to perform a search for specific events you can also use the [search criteria approach](/logs#get-logs-by-search-criteria), which is also the one used by the Management Dashboard.
+
+**Limitation**: When you query for logs with the [list or search logs](/api/v2#!/Logs/get_logs) endpoint, you can retrieve a maximium of 100 logs per request.
 
-#### Retrieving logs from the Management API
+### Get Logs By Checkpoint
```
Comment (machuga) [inline articles/logs/index.md]:
```suggestion
### Get Logs by Checkpoint
```

File: articles/logs/index.md [hunk 6]
```diff
@@ -47,14 +63,54 @@ Enterprise | 30 days
...
 
-You can use the Management API v2 retrieve your logs. There are the two available endpoints, each providing slightly different quantities of information:
+- `from`: Log Event Id to start retrieving logs. You can limit the amount of logs using the take parameter. 
+- `take`: The total amount of entries to retrieve when using the from parameter.
 
-* [/api/v2/logs](/api/v2#!/Logs/get_logs): Retrieves log entries that match the provided search criteria. If you do not provide any search criteria, you will get a list of all available entries;
-* [/api/v2/logs/{id}](/api/v2#!/Logs/get_logs_by_id): Retrieves the single log entry associated with the provided ID.
+Important: When fetching logs by checkpoint, the `q` or any other parameter other than `from` and `take` will be ignored. Also the order by date is not guaranteed.
+
+### Get logs by Search Criteria
```
Comment (machuga) [inline articles/logs/index.md]:
```suggestion
### Get Logs by Search Criteria
```

File: articles/logs/index.md [hunk 7]
```diff
@@ -47,14 +63,54 @@ Enterprise | 30 days
...
+If you get the error `414 Request-URI Too Large` this means that your query string is larger than the supported length. In this case, refine your search.
+
+## Other Log Endpoints
+
+As an alternative or complement to retrieving logs by checkpoint or search criteria using the [/api/v2/logs](/api/v2#!/Logs/get_logs) endpoint, you can also use the following endpoints to look for logs:
+
+* [/api/v2/logs/{id}](/api/v2#!/Logs/get_logs_by_id): Retrieves the single log entry associated with the provided log id.
+* [/api/v2/users/{user_id}/logs](/api/v2#!/Users/get_logs_by_user): Retrieves every log event for a specific user id.
 
 ## Log data event listing
```
Comment (machuga) [inline articles/logs/index.md:119]:
```suggestion
## Log Data Event Listing
```

File: articles/logs/query-syntax.md (hunk 3 above)
Comment (machuga) [inline articles/logs/query-syntax.md]:
```suggestion
The [Management API logs endpoint](/api/v2#!/Logs/get_logs) allows to retrieving logs either by checkpoint (last retrieved log id) or by search criteria. We highly recommend to use the checkpoint approach to export logs to the external system of your choice and perform any search or analysis there, as logs stored in our system are subject to [the retention period](/logs#how-long-is-log-file-data-available). You can use any of the [Export Auth0 logs to an external service](/extensions#export-auth0-logs-to-an-external-service) extensions to export the logs to the system of your choice (like Sumo Logic, Splunk or Loggly).
```

File: articles/logs/query-syntax.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+For eventual searches that you require to perform on the Manage Dashboard or via the Management API, you can use the syntax described in this article to find your logs.
```
Comment (machuga) [inline articles/logs/query-syntax.md]:
```suggestion
When searching for events in the Manage Dashboard or via the Management API, you can use the syntax described in this article to find your logs.
```

File: articles/logs/query-syntax.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+When you query for logs with the [list or search logs]() endpoint, you can retrieve a maximium of 100 logs. Additionally, you may only paginate through up to 1,000 search results. If would like to receive more logs, please retrieve your logs [by checkpoint] retrieval.
+
+If you get the error `414 Request-URI Too Large` this means that your query string is larger than the supported length. In this case, refine your search.
+
+## Search Engine Migration
+
+We are currently migrating our logs search engine to provide customers with the most scalable, robust and fastest search experience. All tenants in the cloud US region were migrated to the latest engine (v3) starting November 15, 2018. Tenants in other cloud regions are still on the previous engine (v2) and will be notified on the migration soon.
+
+### Search Engine v3 Breaking changes
```
Comment (machuga) [inline articles/logs/query-syntax.md]:
```suggestion
### Search Engine v3 Breaking Changes
```

**Review by machuga — DISMISSED** [review]

File: articles/logs/query-syntax.md (hunk 3 above)
Comment (solepano) [reply articles/logs/query-syntax.md]: the section was removed and added similar paragraph in /logs article

File: articles/logs/query-syntax.md (hunk 8 above)
Comment (solepano) [reply articles/logs/query-syntax.md]: section was removed and similar paragraph added to /logs article

**Comment by jeffreylees** [comment]
@d60402 @machuga are we good now?

**Comment by d60402** [comment]
> @d60402 @machuga are we good now?

@jeffreylees This needs to be addressed: https://auth0.slack.com/archives/CEF1UL7UP/p1547569245105800?thread_ts=1546976711.077700&cid=CEF1UL7UP

**Comment by jeffreylees** [comment]
OK, so does anyone object to stripping out the exclusive syntax examples (curly brackets) since that isn't working at this time?

**Comment by machuga** [comment]
@jeffreylees and @d60402 this is addressed here: https://github.com/auth0/docs/pull/7134/files#diff-7e866cd3e303f65bf2dbd80c79621ebbR127 and examples are given above. We can include more if preferred. Ranges otherwise still perform as they did before when not using dates.

**Comment by d60402** [comment]
> @jeffreylees and @d60402 this is addressed here: https://github.com/auth0/docs/pull/7134/files#diff-7e866cd3e303f65bf2dbd80c79621ebbR127 and examples are given above. We can include more if preferred. Ranges otherwise still perform as they did before when not using dates.

@machuga This is not what @jeffreylees was referring to. It's about exclusive {} date range not working as discussed in https://auth0.slack.com/archives/CEF1UL7UP/p1547569245105800?thread_ts=1546976711.077700&cid=CEF1UL7UP

**Comment by machuga** [comment]
@d60402 Can you link to where we recommend `{}`? I'm not seeing it.

**Comment by d60402** [comment]
> @d60402 Can you link to where we recommend `{}`? I'm not seeing it.

@machuga ...

- https://github.com/auth0/docs/pull/7134/files#diff-7e866cd3e303f65bf2dbd80c79621ebbR88
- https://github.com/auth0/docs/pull/7134/files#diff-7e866cd3e303f65bf2dbd80c79621ebbR90
- https://github.com/auth0/docs/pull/7134/files#diff-7e866cd3e303f65bf2dbd80c79621ebbR93
- https://github.com/auth0/docs/pull/7134/files#diff-7e866cd3e303f65bf2dbd80c79621ebbR106
- https://github.com/auth0/docs/pull/7134/files#diff-7e866cd3e303f65bf2dbd80c79621ebbR127

**Comment by machuga** [comment]
@d60402 I see no references in those links to using `{` at the start of a date range (currently the more problematic approach). We can *add* docs to ask people to avoid using `{` or `}` unless using `*`, if that helps. `{}` still performs as expected with integers (log_id, user_id, etc)

**Comment by machuga** [comment]
@d60402 I added some more guidance in the docs. @jeffreylees As long as Daryl doesn't have any further objections I think we're ready to roll this out. Can you also add the Logs page to the left sidebar?

**Comment by d60402** [comment]
@machuga, @jeffreylees lgtm

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**